vmck/api.py

```diff
@@ -23,6 +29,28 @@ def home(request):
     })
 
 
+def create_token(request):
+    auth_header = request.headers['Authorization']
+    token_type, _, credentials = auth_header.partition(' ')
+
+    username, password = base64.b64decode(credentials).decode('utf-8').split(':')
+    # TODO if password is not correct return failure
```

Comment:
Please implement this too, with a test. We can simply return a `JsonResponse` with 401 status code and `{'error': "Invalid credentials"}` as body.